diff --git a/Gemfile.template b/Gemfile.template
index abc123..def456 100644
--- a/Gemfile.template
+++ b/Gemfile.template
@@ -42,6 +42,7 @@ gem "flores", "~> 0.0.8", :group => :development
 gem "json-schema", "~> 2", :group => :development
 gem "logstash-devutils", "~> 2.6.0", :group => :development
 gem "rack-test", :require => "rack/test", :group => :development
+gem "sinatra", "4.1.0", :group => :development
 gem "rspec", "~> 3.5", :group => :development
 gem "webmock", "~> 3", :group => :development
 gem "simplecov", "~> 0.22.0", :group => :development
