--- a/Gemfile.template
+++ b/Gemfile.template
@@ -38,6 +38,7 @@ gem "rubyzip", "~> 1", :group => :build
gem "stud", "~> 0.0.22", :group => :build
# (https://git.ruby-lang.org/ruby.git/commit/?h=ruby_3_2&id=05caafb4731c796890027cafedaac59dc108a23a)
# note that the reason to use 1.7.0 is due to https://github.com/logstash-plugins/logstash-integration-aws/issues/28
gem "fileutils", "~> 1.7"
+gem "sinatra", ">= 4.1.0" # Security fix for GHSA-hxx2-7vcw-mqr3

gem "rubocop", :group => :development
gem "belzebuth", :group => :development
